### PR TITLE
Update fleet-gitops README.md

### PR DESCRIPTION
**Pull request by mike-j-thomas**
Closes https://github.com/fleetdm/fleet/issues/21938

### CODE REVIEW

**Comment by mike-j-thomas** [comment]
Hey, @getvictor, please can you confirm that the command I included for finding gitops documentation in fleetctl is correct?

**Comment by getvictor** [comment]
> Hey, @getvictor, please can you confirm that the command I included for finding gitops documentation in fleetctl is correct?

It is `fleetctl gitops --help`

I recommend putting this at the bottom of the README. Calling `fleetctl gitops` directly is expert usage and you have to know what you're doing.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 This is the starter repository for using [Fleet](https://fleetdm.com) with a GitOps workflow.
 
-How to set up your GitOps workflow:
+> For GitOps workflow documentation, run `fleetctl gitops --help` from fleetctl.
```
Comment (mike-j-thomas) [inline README.md:5]:
```suggestion
> Calling 'fleetctl gitops' directly is only recommended for experienced users. For documentation, run `fleetctl gitops --help`.
```

**Review by getvictor — APPROVED** [review]

**Comment by mike-j-thomas** [comment]
Thanks, @getvictor. Should I be concerned about that failing check before I merge this?

**Comment by getvictor** [comment]
> Thanks, @getvictor. Should I be concerned about that failing check before I merge this?

No. It always fails because we don't have this repo connected to a real Fleet deployment. We have an issue to fix it.

**Pull request merged**